sechub-web-ui/.env

```diff
@@ -0,0 +1,9 @@
+# This .env is the default environmentfile
+# Variables need to start with Vite_API
+VITE_API_HOST=http://localhost:3000
+
+# For local development create an env.local with the desired variables
+# it will be ignored by git 
+VITE_API_LOCAL_DEV=false
+VITE_API_USER='example-test-user'
+VITE_API_PASSWORD='example-api-token'
```

Comment:
```suggestion
# This .env is the default environmentfile

# For local development create an env.local with the desired variables
# it will be ignored by git.
# Local development variables need to start with VITE_API
VITE_API_HOST=http://localhost:3000
VITE_API_LOCAL_DEV=false
VITE_API_USER='example-test-user'
VITE_API_PASSWORD='example-api-token'
```

Just a remark: is `VITE_API_*` really good here?
Would `LOCAL_DEV_*` not be better? (loose coupling, dev proxy could be changed without naming issue etc.)

But if names are changed, this must be also in changed in docs and env files. So maybe a follow up issue would be better.